Gemfile

```diff
@@ -4,9 +4,11 @@ source 'http://rubygems.org'
 
 gemspec
 
+gem 'activesupport', '>= 4.2', '< 7'
 gem 'appraisal'
 gem 'bump', '~> 0.6.1'
 gem 'coveralls', require: false
+gem 'exception_handling'
```

Comment:
We don't want this public gem depending on `exception_handling`! That's too onerous of a dependency--and the reason for TECH-4853. Instead, you'll need to flatten that code inline here.